File: src/main/java/ru/dkolmogortsev/messages/TimeEntryDeleted.java
package ru.dkolmogortsev.messages;

import griffon.core.event.Event;

/**
 * Created by dkolmogortsev on 3/11/17.
 */
public class TimeEntryDeleted extends Event
{

    public TimeEntryDeleted(long entryId)
    {
        super(entryId);
    }

    public long getEntryId()
    {
        return (long)getSource();
    }
}

File: src/main/kotlin/ru/dkolmogortsev/messages/TimeEntryDeleted.kt
package ru.dkolmogortsev.messages

import griffon.core.event.Event

/**
 * Created by dkolmogortsev on 3/11/17.
 */
class TimeEntryDeleted(entryId: Long) : Event(entryId) {

    val entryId: Long
        get() = getSource() as Long
}
